mat64/solve.go

```diff
@@ -11,15 +11,15 @@ import (
 	"github.com/gonum/matrix"
 )
 
-// Solve solves a minimum-norm solution to a system of linear equations defined
-// by the matrices a and b. If a is singular or near-singular a Condition error
+// Solve finds a minimum-norm solution to a system of linear equations defined
+// by the matrices A and b. If A is singular or near-singular a Condition error
```

Comment:
I know that we distinguish between the two but it is true that I'm not sure which one we choose when either is possible (as in this case). I went with the common linear algebra notation because SolveLU et al use it (true, they don't have the matrix explicitly among the arguments) but also because `Dense.Solve` has `... a and b. ...` while `Vector.Solve` has `... A and B. ...`.

Do you mean that it should be: small a and b in this paragraph and in the comment for `Vector.Solve`, and capital A and B in the next paragraph?